<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1891</width>
      <height>821</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::TabShape::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>TestBench</string>
     </attribute>
     <widget class="QWidget" name="Container_6" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>191</width>
        <height>251</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>191</width>
         <height>231</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="afficheur_19" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_65">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Vide primaire</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_72">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>28</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>mbar</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_17">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="afficheur_24" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_73">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Vide primaire</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_74">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>28</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>mbar</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_18">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="afficheur_25" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_75">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>85</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Vide secondaire</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_76">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>28</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>mbar</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_19">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_80">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>37</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pompe</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Container_7" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>191</width>
         <height>161</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="afficheur_22" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_70">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Courant</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_77">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>15</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ÂµA</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_20">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="afficheur_26" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_78">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tension</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_79">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>28</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>KV</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_21">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_83">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>37</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Puller</string>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="listWidget_testBench">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>1471</width>
        <height>461</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
     </widget>
     <widget class="QWidget" name="Container_11" native="true">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>191</width>
         <height>161</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="afficheur_23" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_71">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Courant</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_81">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>15</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ÂµA</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_22">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="afficheur_27" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_82">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tension</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_84">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>28</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>KV</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_23">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_85">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Electrodes</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Container_12" native="true">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>20</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="afficheur_28" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_86">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>DÃ©bit</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_87">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>32</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>SCCM</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_24">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_90">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gaz</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Paramtres">
     <attribute name="icon">
      <iconset theme="applications-development"/>
     </attribute>
     <attribute name="title">
      <string>ParamÃ¨tres</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>621</width>
        <height>284</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>621</width>
         <height>264</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>99</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Chemin du dossier</string>
        </property>
       </widget>
       <widget class="QWidget" name="numInput_14" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>149</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_56">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>149</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>FrÃ©quence d'enregistrement</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_57">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>19</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Sec</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="spinBox_logFrequency">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimum">
          <double>0.050000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="numInput_13" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>148</width>
          <height>61</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_52">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>139</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Limite de taille par fichiers</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_fileSizeLimit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_55">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>18</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Mb</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>197</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Activer l'enregistrement des donnÃ©es</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_LogOnOff">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>On</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_logPath">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>591</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Enregistrement des donnÃ©es</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>621</width>
        <height>284</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>621</width>
         <height>264</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="label_51">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ports disponibles</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Refresh">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>30</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraichir</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_PortList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>31</y>
          <width>481</width>
          <height>221</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Appareils connectÃ©s</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
